src/components/DatePicker/RangePicker.jsx

```diff
@@ -377,8 +447,8 @@ const FakeInput = styled(Flex)(({ expanded, disabled, theme }) =>
       disabled
         ? theme.colors.inputBorderGray
         : expanded
-        ? theme.colors.accent
-        : theme.colors.inputBorderGray
+          ? theme.colors.accent
+          : theme.colors.inputBorderGray
```

Comment:
Avoid using nested ternary operators.

Reply:
Yes, but it's better to consider what is in scope / out of scope for a PR